src/sinks/new_relic/config.rs

```diff
@@ -134,6 +134,10 @@ impl SinkConfig for NewRelicConfig {
     fn sink_type(&self) -> &'static str {
         "new_relic"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
Same here, can this be `true`? Can NewRelic accept data but then drop it?

Reply:
Looks like this is covered, and I missed it. Will fix.